.env.example

```diff
@@ -16,7 +16,7 @@ MADARA_PRIVATE_KEY=0x00c1cf1490de1352865301bb8705143f3ef938f97fdf892f1090dcb5ac7
 KAKAROT_HTTP_RPC_ADDRESS=0.0.0.0:3030
 ## check `./deployments/katana/deployments.json` after running `make devnet`
 KAKAROT_ADDRESS=
-PROXY_ACCOUNT_CLASS_HASH=0x3010a53967fa04842bcbcb6de8817101f047ef0d074b3eacbe714a3fc42a2eb
+PROXY_ACCOUNT_CLASS_HASH=0x4b9eef81a3f0a582dfed69be93196cedbff063e0fa206b34b4c2f06ac505f0c
```

Comment:
not sure why this was changed, but this should be deterministic, yes?

Reply:
Maybe due to some changes in Kakarot contract it changed